client/README.md

```diff
@@ -1,5 +1,7 @@
 # InfluxDB Client
 
+[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/influxdb/influxdb/client)
```

Comment:
...actually only because the official one looks awful on high-res displays.

But now that you mention it, there is also a SVG version of the official badge now
https://godoc.org/github.com/influxdb/influxdb?status.svg

Prefer this?